### ISSUE TITLE
Environment Canada weather displays wrong times for hourly weather

### ISSUE THREAD

**Issue by briodan** [MichMich/MagicMirror#3080]
When configuring envcanada as the weather source the hourly time is displayed wrong, the temperatures match what is being reported by the weather station in the correct order

![image](https://user-images.githubusercontent.com/41130293/229986454-b5db221a-875d-45e9-9fbe-615428ac7cc6.png)
![image](https://user-images.githubusercontent.com/41130293/229986805-5d7754ec-cd35-49a4-8391-7f68e4353f89.png)

here is the config i'm using
```
		{
			"module": "weather",
			"header": "Current Weather",
			"position": "top_right",
			"config": {
				"location": "Calgary, AB",
				"weatherProvider": "envcanada",
				"siteCode": "s0000047",
				"provCode": "AB",
				"roundTemp": false,
				"type": "hourly",
				"units": "metric",
				"useKmh": false,
				"tempUnits": "metric",
				"windUnits": "metric",
				"updateInterval": 600000,
				"animationSpeed": 1000,
				"timeFormat": 12,
				"showPeriod": true,
				"showPeriodUpper": false,
				"showWindDirection": true,
				"showWindDirectionAsArrow": false,
				"useBeaufort": true,
				"lang": "en",
				"showHumidity": false,
				"showSun": true,
				"degreeLabel": true,
				"decimalSymbol": ".",
				"showIndoorTemperature": false,
				"showIndoorHumidity": false,
				"maxNumberOfDays": 5,
				"maxEntries": 12,
				"ignoreToday": false,
				"fade": false,
				"fadePoint": 0.25,
				"initialLoadDelay": 0,
				"appendLocationNameToHeader": true,
				"calendarClass": "calendar",
				"tableClass": "small",
				"onlyTemp": false,
				"showPrecipitationAmount": false,
				"colored": false,
				"showFeelsLike": true,
				"absoluteDates": false
			}
```

seeing this issue on both a Pi3 and a debian 11.4.0 linux vm, both using the april release, note this issue was also present in the previous release.

**Comment by rejas** [comment]
Thx for the report @briodan can you maybe test my PR if it fixes your problem for real?

**Comment by briodan** [comment]
yeap that did the trick, thanks

![image](https://user-images.githubusercontent.com/41130293/230785902-65016373-ba7f-4093-a16c-2db3ad4ba7d8.png)
![image](https://user-images.githubusercontent.com/41130293/230785925-ade02758-b1af-4272-b0d3-8d69fbca17c1.png)

**Comment by khassel** [comment]
fixed with new release `v2.24.0`
